stories/index.js

```diff
@@ -24,4 +26,24 @@ storiesOf('Image', module).addParameters({
     ],
 }).add('default', () => (
     <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
+));
+
+storiesOf('Tabs', module).addParameters({
```

Comment:
I start to think maybe we should split stories into different files, to avoid merge conflicts.

Reply:
This sounds good, we've split the stories for each existing component and updated the storybook config.js to load them dynamically.

Reply:
This was done by @kumardennis